example/cumulus-tf/orca.tf

```diff
@@ -12,8 +12,8 @@ locals {
 
 # ORCA Module
 module "orca" {
-  count = 1
-  source = "https://github.com/nasa/cumulus-orca/releases/download/v8.0.1/cumulus-orca-terraform.zip"
+  aws_region = var.region
```

Comment:
Is the `count` param here removed because the orca module doesn't support it? Obviously we'd only want 1 Orca instance so at best it's redundant I'd guess.

Reply:
Correct - Orca removed that parameter.